.github/workflows/ubuntu-windows.yml

```diff
@@ -0,0 +1,363 @@
+name: Windows-Ubuntu
+
+on:
+  push:
+    branches:
+      - main
+      - feature/*
+  release:
+    types: [ created ]
+
+env:
+  GITHUB_TOKEN: ${{ github.token }}
+
+jobs:
+
+  build:
+    name: ${{ matrix.os }} shared=${{ matrix.shared }} sirius=${{ matrix.sirius }} xprs=${{ matrix.xprs }}
+    runs-on: ${{ matrix.os }}
+    env:
+      Sirius_Ref: antares_integration
+      XPRS_Path: ${{ github.workspace }}/xpressmp813
+      XPRESS: ${{ github.workspace }}/xpressmp813/bin
+      XPRS_LIB_Path: ${{ github.workspace }}/xpressmp813/lib
+    strategy:
+      matrix:
+        os: [windows-2019, ubuntu-20.04]
+        xprs: [ON, OFF]
+        sirius: [ON, OFF]
+        shared: [ON, OFF]
+        extras: [ON, OFF]
+
+        exclude:
+          - xprs: OFF
+            sirius: OFF
+          - os: windows-2019
+            shared: ON
+          - os: windows-2019
+            shared: OFF
+            extras: OFF
+          - os: ubuntu-20.04
+            shared: OFF
+            extras: ON
+          - os: ubuntu-20.04
+            shared: ON
+            extras: OFF
+        include:
+          - extras: ON
+            java: ON
+            python: ON
+            dotnet: OFF
+          - extras: OFF
+            java: OFF
+            python: OFF
+            dotnet: OFF
+
+
+    steps:
+      - name: set name variables
+        id: names
+        shell: bash
+        run: |
+          REF=${GITHUB_REF#refs/*/}
+          REF_WO_SLASHES=${REF//\//-}
+          WITH_VERSION="_${REF_WO_SLASHES}"
+          SHARED=${{ matrix.shared }}
+          [ $SHARED == "ON" ] && WITH_SHARED="_shared" || WITH_SHARED="_static"
```

Comment:
What's the meaning of this ?

Reply:
if SHARED:
add a suffix "_shared" to the targets
else
add a suffix "_static" to the targets

Reply:
we can remove it or do in another way